### PR TITLE
SCRUM-3644: Reorder logic for filters

### PR DESCRIPTION
**Pull request by abecerra**

### CODE REVIEW

**Review by adamgibs — APPROVED** [review]

File: src/components/dataTable/DataTable.js [hunk 1]
```diff
@@ -96,7 +96,7 @@ const DataTable = ({
   }
 
   // the initial fetch is happening, show spinner, don't render the table yet
-  if (isLoading || isFetching) {
```
Comment (oblodgett) [inline src/components/dataTable/DataTable.js:99]: I believe you want the isFetching because when they hit the second page this needs to turn on.

**Review by oblodgett — CHANGES_REQUESTED** [review]
Has this been tested on all Data Tables on the site?

**Comment by abecerra** [comment]
@oblodgett: without the `isFetching`, the pagination+filter issue is solved. I am not sure why 
I tested in the gene and disease pages (with the disease ribbon)

**Comment by oblodgett** [comment]
We need to have the isFetching so somewhere there we need to fix it... this need to not be a hack fix

**Review by oblodgett — APPROVED** [review]

**Pull request merged**